src/mini_allocator.c

```diff
@@ -337,7 +337,7 @@ static uint64
 mini_num_entries(page_handle *meta_page)
 {
    mini_meta_hdr *hdr = (mini_meta_hdr *)meta_page->data;
-   return hdr->num_entries;
+   return (uint64)hdr->num_entries;
```

Comment:
Shouldn't really matter but just conforming to return type declared for this fn's definition.